### PR TITLE
Gradopaw/lesson 11

### PR DESCRIPTION
**Pull request by akpoik**
@gradopaw

### CODE REVIEW

File: students/kulon_aleksander/lesson_11/filling_in_the_gaps.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+def main():
+    sorted_numbers = []
+
+    parser = argparse.ArgumentParser(description='Filling gaps.')
+
+    parser.add_argument(
+        '--pattern', action="store",
+        dest='pattern',
+        default=pattern,
```
Comment (gradopaw) [inline students/kulon_aleksander/lesson_11/filling_in_the_gaps.py]: ```default``` flag shouldn't be used with ```required``` flag set to true. It hasn't sense😄 Have a look at this short descriptions: https://docs.python.org/2/library/argparse.html#argparse.ArgumentParser.add_argument

File: students/kulon_aleksander/lesson_11/filling_in_the_gaps.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        required='True')
```
Comment (gradopaw) [inline students/kulon_aleksander/lesson_11/filling_in_the_gaps.py]: For ```required``` flag bool is expected not string

File: students/kulon_aleksander/lesson_11/selective_copy.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+new_folder = 'new_folder'
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Selective copy.')
+
+    parser.add_argument(
+        '-d', action="store",
+        dest='new_folder',
+        required='True')
```
Comment (gradopaw) [inline students/kulon_aleksander/lesson_11/selective_copy.py]: bool instead of string

**Comment by akpoik** [comment]
@gradopaw

**Pull request merged**